server/stream.go

```diff
@@ -1434,6 +1452,11 @@ func (mset *stream) update(config *StreamConfig) error {
 
 // Update will allow certain configuration properties of an existing stream to be updated.
 func (mset *stream) updateWithAdvisory(config *StreamConfig, sendAdvisory bool) error {
+	_, jsa, err := mset.acc.checkForJetStream()
```

Comment:
Can we assume mset.acc non nil here? Can't remember off top of my head.

Reply:
There is an mset.account() accessor function.  Better to use that and check for nil return?

Reply:
I think so.